admin/templates/login.html

```diff
@@ -0,0 +1,67 @@
+{% load render_bundle from webpack_loader %}
```

Comment:
@cwisecarver, is there a way to do this without including the rest of base.html's page content?